packages/jsapi-types/src/dh.types.ts

```diff
@@ -82,6 +82,20 @@ export interface VariableDefinition<T extends string = string> {
   id?: string;
 }
 
+export interface JsWidget extends Evented {
+  getDataAsBase64: () => string;
+  getDataAsU8: () => string;
```

Comment:
Should be Uint8Array.
Either way we should be replacing this with the automatic typings, so let's not worry too much about the manual typings right now as long as it's working correctly.

Reply:
Good point. I don't even use the U8, but figured I'd add it in case we do use it before we get the automatic typings